templates/gallery/gallery-inner.amp.json

```diff
@@ -0,0 +1,77 @@
+{
+  "gallery": {
+    "head-tag": {
+      "title": "Gallery Template",
+      "canonical-path": "templates/gallery-inner.amp",
+      "extensions": [
+        "amp-iframe",
+        "amp-carousel",
+        "amp-sidebar"
+      ],
+      "css-path": "templates/gallery/page.css",
+      "font-stylesheets": [
+        "https://fonts.googleapis.com/css?family=Poppins:300,400,600",
+        "https://fonts.googleapis.com/css?family=Oswald:400"
+      ]
+    },
+    "hero": {
+      "heading": "Living Tree",
+      "subheading": "<time datetime=\"2017-08-24\">Sept 24</time>-<time datetime=\"2017-08-29\">29</time> / Anna Moss",
+      "button": {
+        "text": "Buy Tickets"
+      },
+      "carousel": {
+        "height": 316,
+        "images": [{
+          "width": 210,
+          "height": 316,
+          "src": "../../img/gallery/court.jpg",
+          "2x": "../../img/gallery/court@2x.jpg",
+          "alt": "outdoor basketball court"
+        }, {
+          "width": 361,
+          "height": 316,
+          "src": "../../img/gallery/corner.jpg",
+          "2x": "../../img/gallery/corner@2x.jpg",
+          "alt": "corner of a building"
+        }, {
+          "width": 211,
+          "height": 316,
+          "src": "../../img/gallery/lamp.jpg",
+          "2x": "../../img/gallery/lamp@2x.jpg",
+          "alt": "pink lamp on a blue background"
+        }, {
+          "width": 316,
+          "height": 316,
+          "src": "../../img/gallery/plant.jpg",
+          "2x": "../../img/gallery/plant@2x.jpg",
+          "alt": "house plant on chair on yellow background"
+        }, {
+          "width": 517,
+          "height": 316,
+          "src": "../../img/gallery/stripe.jpg",
+          "2x": "../../img/gallery/stripe@2x.jpg",
+          "alt": "striped building"
+        }, {
+          "width": 473,
+          "height": 316,
+          "src": "../../img/gallery/hall.jpg",
+          "2x": "../../img/gallery/hall@2x.jpg",
+          "alt": "colourful hallway"
+        }]
+      },
+      "content": "<blockquote>Justo libero ut nunc consequat massa. Vitae et nulla. Mus arcu in. Tellus at ut. Ac mi varius ut.</blockquote>"
+    },
+    "map": {
+      "heading": "Visit the exhibition from <time datetime=\"2017-08-24\">Sept 24</time>-<time datetime=\"2017-08-29\">29</time>",
+      "subheading": "123 St Name <br/>City, State 12345",
```

Comment:
This is not subheading ,this belongs in an address tag, and please try not to use BRs